cfpb/consumerfinance.gov
# Fix mixed declaration warnings in SCSS

## Changes

- Change order of CSS declarations to fix Sass deprecation warnings.


## How to test this PR

1. `yarn build` shouldn't have any deprecation warnings for mixed declarations. Check the touched apps for any visual changes. They should be unchanged to prod.
